docs/interactionmanager.md

```diff
@@ -37,6 +37,157 @@ By default, queued tasks are executed together in a loop in one `setImmediate` b
 
 ---
 
+## Example
+
+### Basic
+
+```SnackPlayer name=InteractionManager%20Function%20Component%20Basic%20Example&supportedPlatforms=ios,android
+import * as React from 'react';
+import {
+  Alert,
+  Animated,
+  InteractionManager,
+  Platform,
+  StyleSheet,
+  Text,
+  View,
+} from 'react-native';
+
+const instructions = Platform.select({
+  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
+  android:
+    'Double tap R on your keyboard to reload,\n' +
+    'Shake or press menu button for dev menu',
+});
+
+const useMount = (func) => React.useEffect(() => func(), [])
+
+const useFadeIn = (duration = 5000) => {
+  const [opacity] = React.useState(new Animated.Value(0));
+
+  // Running the animation when the component is mounted
+  useMount(() => {
+    // Animated.timing() create a interaction handle by default, if you want to disabled that
+    // behaviour you can set isInteraction to false to disabled that.
+    Animated.timing(opacity, {
+      toValue: 1,
+      duration,
+    }).start();
+  });
+
+  return opacity;
+};
+
+function Ball({ onShown }) {
```

Comment:
It would be nice if you could continue using arrow function over `function` keyword throughout whole example for the consistency.

Reply:
I don't really recommend use arrow function on function components.

Reply:
May I ask you why?

Reply:
@Simek  you need to do a extra step adding the displayName to the arrow function making easy to debug through react-dev-tools

Reply:
It's not true right now, but there was a problem with `displayName` and arrow function a few years ago (when ES2015/ES6 was not so widely used). 

Please check out this issue https://github.com/facebook/react-devtools/issues/267#issuecomment-283793517 and especially Gaearon comments.

Correct `Babel` transformation preset is included in `Metro` RN presets module for a long time: https://github.com/facebook/metro/blob/master/packages/metro-react-native-babel-preset/package.json.

Reply:
@Simek Ok, let me rewrite the code to use arrow functions.

Reply:
BTW. I was 99% sure but to be certain I have tested this in https://codesandbox.io/ (there is a build in Dev Tools panel) and I can confirm that there are no issues with functional Component name even with inline `export`:
```
export default TestApp = () => <jsx />;
```

Reply:
@Simek but that is a little bit confusing for new developers and is not common used. I think now looks good.